flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowFormatFactoryTest.java

```diff
@@ -91,6 +103,18 @@ public void testJsonSchema() {
 		testJsonSchemaDeserializationSchema(properties);
 	}
 
+	@Test
+	public void testJsonSchemaIgnoreParseErrors() {
```

Comment:
I think `testSchemaIgnoreParseErrors()` is enough to cover the bug. We can't permute all possible combination of configurations.

Reply:
ok , i will delete this method.

Reply:
i have delete this test method.